integration-tests/oidc-wiremock/src/main/resources/application.properties

```diff
@@ -3,5 +3,14 @@ quarkus.oidc.auth-server-url=${keycloak.url}/realms/quarkus/
 quarkus.oidc.client-id=quarkus-app
 quarkus.oidc.credentials.secret=secret
 quarkus.oidc.token.principal-claim=email
+quarkus.oidc.authentication.scopes=profile,email,phone
```

Comment:
@cemnura please drop lines 6-10 (just adjust the test check where it verifies the redirect value)